game_calls.cpp

```diff
@@ -1,14 +1,25 @@
-#include "game_calls.h"
```

Comment:
Yeah not too sure yet on how to order includes. The tutorial I was following suggested including my own ".h"files first

Reply:
Ideally the ordering shouldn't matter, but some headers change based on macros defined when it is included, some others define macros, and that's where it gets hairy. This is why we have all our `#DEFINE`s in one place.

IMHO system headers first makes sense because I follow the "least to most dependent" or "farthest away to closest" in the ordering of headers. (Core first, then Engine, then Game, for example.)